js/main.js

```diff
@@ -1,2 +1,29 @@
 /* global data */
 /* exported data */
+function imgUrlInput(event) {
+  var imgsrc = event.target.value;
+  $imageDisplay.setAttribute('src', imgsrc);
+}
+function formJournalSubmit(event) {
+  var newObj = {};
+  event.preventDefault();
+  newObj.title = $formJournal.elements.title.value;
+  newObj.imgsrc = $formJournal.elements.photoUrl.value;
+  newObj.notes = $formJournal.elements.notes.value;
+  newObj.nextEntryId = data.nextEntryId;
+  data.entryId = (data.nextEntryId) + 1;
```

Comment:
You're assigning a value to a property named `netryId`, but there is no such property in `data`. You want to increment the value of the `nextEntryId` property by 1. `data.nextEntryId++`